src/test/java/core/basesyntax/LambdaCalcTest.java

```diff
@@ -0,0 +1,245 @@
+package core.basesyntax;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+class LambdaCalcTest {
+    private static LambdaCalc lambdaCalc;
```

Comment:
Not informative naming. 
```suggestion
class CalculatorTest {
    private static CalculatorTest calculatorTest;
```

Reply:
done, renamed to more informative naming